### PR TITLE
OPENEUROPA-759: URI is malformed when property "base_url" has a predefined path (#23)

### PR DESCRIPTION
**Pull request by dxvargas**

### CODE REVIEW

File: src/Cas/Protocol/AbstractCasProtocol.php [hunk 1]
```diff
@@ -210,10 +210,12 @@ public function get($name, array $query = [])
             )
         );
 
-        return $this->getUriFactory()
+        $uri = $this->getUriFactory()
             ->createUri($properties['base_url'])
-            ->withPath($properties['protocol'][$name]['path'])
             ->withQuery(http_build_query($query));
+        $uri = $uri->withPath($uri->getPath() . $properties['protocol'][$name]['path']);
```
Comment (netlooker) [inline src/Cas/Protocol/AbstractCasProtocol.php]: Is there a reason to split this $uri into parts overridden one after another?

Comment (drupol) [reply src/Cas/Protocol/AbstractCasProtocol.php]: I don't get it... Maybe because i'm driving :)

Comment (drupol) [reply src/Cas/Protocol/AbstractCasProtocol.php]: I would rather not overwrite variable and do:

```php
return $uri->withPath($uri->getPath() . $properties['protocol'][$name]['path']);
```

Can we please do a PR and fix it ? or we can commit it directly to master ?

Comment (dxvargas) [reply src/Cas/Protocol/AbstractCasProtocol.php]: @drupol we can change this. Please don't commit it in master, just crate a PR and proceed as usual.

**Pull request merged**